### DISCUSSION TITLE
LaserGRBL software stops working in the middle of engraving

### DISCUSSION THREAD

**Discussion by AmericanBuilt** [arkypita/LaserGRBL#2474] [General]
I am using a "LaserTree K1 Mini" diode laser unit and currently running "LaserGRBL V.7.14.0"
The last several engraving projects (small in size - 90mm x 90mm) have been stopping without completing the project with the following message: "E (26) efuse: range of data does not match coding scheme"
I have run the same image using LightBurn and have no problems with the laser stopping so I'm fairly convinced it is the LaserGRBL software.
![Screenshot 2024-12-16 191842](https://github.com/user-attachments/assets/0dbbc2dd-c6e4-4661-97ff-db1244d05ecb)

Not sure where to go from here, I appreciate any help.

**Comment by masawee** [comment]
i have similar or not problem, i load focus test file G-code file focus.nc
then open file and run file, laser working well and run program but all windows10 freexe not response anythink, mouse no move, no can stop laser, no can do anythink all windows has stopping work, ctrl-alt-del not working no can do anything. need shutdown all computer and start again. but all files can open and run but then all computer freeze sure. ewery file what try burn. what hell is  wrong ?
